src/webui/www/private/views/preferences.html

```diff
@@ -199,9 +199,16 @@
     </fieldset>
 
     <fieldset class="settings">
+        <legend>
+            <input type="checkbox" id="autorunOnTorrentAddedCheckbox" onclick="qBittorrent.Preferences.updateAutoRunOnTorrentAdded();" />
```

Comment:
This doesn't look exactly like in the GUI, are you able to modify it?

Reply:
> This doesn't look exactly like in the GUI

Even previously it didn't look exactly like GUI, did it?


> are you able to modify it?

I would prefer to limit myself to minimal changes in this area.

Reply:
Well, now I see what you mean.
Should be fixed now.